3.optimizing_CP/run_sam/README.md

```diff
@@ -0,0 +1,4 @@
+Using SAM2 to track objects (Nuclei) over time in a live-cell time-lapse movie of HeLa cells.
+
+## Tracking of a single FOV of multiple cells
+![Tracking of a single FOV of multiple cells](./masks/out.gif)
```

Comment:
I don't see this being committed in this PR, so I don't think this will work once merged. Recommend double checking this.

Reply:
adjusted .gitignore